generators/client-2/templates/src/main/webapp/app/account/reset/finish/_reset-finish.component.ts

```diff
@@ -0,0 +1,53 @@
+import {Component, OnInit, Inject, Renderer, ElementRef} from '@angular/core';
+import { TranslatePipe } from '../../../shared/translate.pipe';
+
+@Component({
+    selector: 'password-reset-finish',
```

Comment:
actually `password-reset-finish` is clearer. Ideally the file should have been named as such as well